errors/format_test.go

```diff
@@ -352,10 +354,10 @@ func TestFormatGeneric(t *testing.T) {
 		},
 	}
 
-	for s := range starts {
-		err := starts[s].err
-		want := starts[s].want
-		testFormatCompleteCompare(t, s, err, "%+v", want, false)
+	for i := range starts {
+		err := starts[i].err
+		want := starts[i].want
+		testFormatCompleteCompare(t, i, err, "%+v", want, false)
```

Comment:
Curious about this one: any context on why we do `for i := range starts` instead of `for _, start := range starts {`? More of a stylistic item but I've leaned towards the explicit form where we show that the range clause is actually 2 values

Reply:
I've just renamed the variable based on https://dave.cheney.net/practical-go/presentations/qcon-china.html#_use_a_consistent_naming_style

---

Also, it can't be `for _, start := range starts {` cause we need to pass index to the `testFormatCompleteCompare()`.

From a performance perspective, index access is preferable for the slice of big objects - https://go-critic.com/overview.html#rangeValCopy-ref